catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -906,7 +834,24 @@ String getSortKey(String field) {
   }
 
   Stream<String> anyTextFields() {
-    return filteredAnyTextFieldsCache.stream();
+    return anyTextFields.stream();
+  }
+
+  private Set<String> getAnyTextFields() {
+    String property =
+        AccessController.doPrivileged(
+            (PrivilegedAction<String>) () -> System.getProperty(SOLR_ANYTEXT_FIELDS, ""));
+
+    List<String> fields;
+    if (property.isEmpty()) {
+      fields = Arrays.asList(Metacard.METADATA, Metacard.TITLE, Metacard.DESCRIPTION);
+    } else {
+      fields = Arrays.asList(property.split(","));
+    }
+
+    return fields.stream()
```

Comment:
❓  Should we filter blank strings to account for redundant commas in the system property?

Reply:
❓ Would it be worth adding a test for that case as well?

Reply:
Probably not a bad idea.  I can also look at removing whitespace padding too if it is present.